packages/messaging-api-messenger/src/MessengerClient.ts

```diff
@@ -44,8 +44,10 @@ function handleError(
 ): never {
   if (err.response && err.response.data) {
     const error = get(err, 'response.data.error', {});
-    const msg = `Messenger API - ${error.code} ${error.type} ${error.message}`;
-    throw new AxiosError(msg, err);
+    if (error) {
```

Comment:
default value of `error` after lodash get is `{}` will always pass `if` condition